src/layer/VideoOverlay.js

```diff
@@ -32,7 +32,11 @@ export var VideoOverlay = ImageOverlay.extend({
 
 		// @option loop: Boolean = true
 		// Whether the video will loop back to the beginning when played.
-		loop: true
+		loop: true,
+
+		// @option saveAspectRatio: Boolean = true
+		// Whether the video will save aspect ratio after the projection.

```

Comment:
Can you please change the name to `keepAspectRatio` here? Also adding a note about browser compatibility would be good.

Reply:
where need i to add a note about browser compatibility?

Reply:
In the these comments. Something like  

```
// Whether the video will keep its original aspect ratio. Setting it to `false`
// only has effect on browsers which implement the 
// [`object-fit` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit)".
```